src/gulp-jsdoc.js

```diff
@@ -73,12 +73,14 @@ export function jsdoc(config, done) {
                 
                 let args = ['-c', tmpobj.name];
                 
-                // Config + ink-docstrap if user did not specify their own layout
-                if(!jsdocConfig.templates || 
-                    !jsdocConfig.templates.default || 
-                    !jsdocConfig.templates.default.layoutFile){
+                // Config + ink-docstrap if user did not specify their own layout or template
+                if((!jsdocConfig.opts ||
```

Comment:
if((!(jsdocConfig.opts && jsdocConfig.opts.template) &&
!(jsdocConfig.templates && jsdocConfig.templates.default && jsdocConfig.templates.default.layoutFile)) { 

which can then be simplified via lodash: https://lodash.com/docs#has 
if(!_.has(jsdocConfig, 'opts.template) && !_.has(jsdocConfig, 'templates.default.layoutFIle')) {...
or
if(!(_.has(jsdocConfig, 'opts.template) || _.has(jsdocConfig, 'templates.default.layoutFIle')) {...

Just to make it more readable